next-lesson/setup.md

```diff
@@ -0,0 +1,11 @@
+<div style="background: red; color: black;">FIXME</div>
+
+Please delete the contents of this file and fill it with your own
+
+## Expected contents
+
+A list of the requirements.
+
+## Example
+
+- For most users we recommend that you use `conda` to install the requirements for the workshop.
```

Comment:
Add information that the setup should cover 3 major platforms (WIndows, MacOS and Linux).

Reply:
it is added.